spec/features/assignments/index_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'viewing the calendar' do
+  let(:roster) { create :roster }
+  let(:user) { create :user, rosters: [roster] }
+  before :each do
+    Timecop.freeze Date.new(2017, 8, 14)
+    set_current_user(user)
+  end
+
+  it 'highlights today' do
+    visit roster_assignments_url(roster)
+    expect(page).to have_selector('td.cal-cell.current-day', text: '14')
+  end
+  context 'assignment belongs to user' do
+    it 'appears highlighted yellow for your assignment' do
+      create :assignment, start_date: 2.days.ago, end_date: 2.days.since,
+                          user: user, roster: roster
+      visit roster_assignments_url(roster)
+      expect(page).to have_selector('td .cal-event.assignment-user', count: 5)
+    end
+  end
+
+  context 'assignment does not belong to user' do
+    it 'appears highlighted blue for other assignments' do
```

Comment:
Should `it` say 'highlighted blue'? As far as I can tell, the other assignments are highlighted differently than the current user's assignments by `assignment-user` versus `assignment`. Nothing about the color. Just something I noticed.